src/App.jsx

```diff
@@ -14,70 +16,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [thisGood, setThisGood] = useState('Jam');
+  const NoSelected = 'No goods selected';
+  const clearGoodsState = () => {
+    setThisGood(null);
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const checkToRemove = (good) => {
+    if (thisGood !== good) {
+      setThisGood(good);
+    } else {
+      clearGoodsState();
+    }
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {thisGood ? `${thisGood} is selected` : NoSelected}
+        {thisGood !== null && (
```

Comment:
Checking for null is obviously not a good idea, think about how it can be rewritten